peer/pendingheap/list.go

```diff
@@ -48,6 +53,15 @@ func Capacity(capacity int) ListOption {
 	}
 }
 
+// Seed specifies the random seed to use for shuffling peers
+//
+// Defaults to approximately the process start time in nanoseconds.
```

Comment:
Probably a little more elaborate than necessary: Defaults to the time in nanoseconds.